powerlevel9k.zsh-theme

```diff
@@ -1298,6 +1298,8 @@ powerlevel9k_prepare_prompts() {
   # Reset start time
   _P9K_TIMER_START=0xFFFFFFFF
 
+  setopt PROMPT_SUBST
+
   if [[ "$POWERLEVEL9K_PROMPT_ON_NEWLINE" == true ]]; then
     PROMPT='$(print_icon 'MULTILINE_FIRST_PROMPT_PREFIX')%f%b%k$(build_left_prompt)
```

Comment:
You are referring to the multiple single quotes, right? I have done such quoting style on other places and it seems to work correctly.. I could straighten it a bit by setting `MULTILINE_FIRST_PROMPT_PREFIX` in double quotes..